namespaces/live-1.cloud-platform.service.justice.gov.uk/manage-soc-cases-preprod/00-namespace.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: manage-soc-cases-preprod
+  labels:
+    cloud-platform.justice.gov.uk/is-production: "false"
+    cloud-platform.justice.gov.uk/environment-name: "preprod"
+  annotations:
+    cloud-platform.justice.gov.uk/business-unit: "HMPPS"
+    cloud-platform.justice.gov.uk/application: "manage-soc-cases"
+    cloud-platform.justice.gov.uk/owner: "Digital Prison Services: dps-hmpps@digital.justice.gov.uk"
+    cloud-platform.justice.gov.uk/source-code: "https://github.com/ministryofjustice/manage-soc-cases.git"
+    cloud-platform.justice.gov.uk/slack-channel: "#pathfinder-service"
```

Comment:
Sorry to be a pain, I should have specified that this needs to be without the leading `#` My bad.

Reply:
No problem. That's updated now.